docs/content_editors/api_guidance_pages.md

```diff
@@ -0,0 +1,90 @@
+[< Back to Navigation](../README.md)
+
+# API Guidance Pages
+
+This document is designed to assist content editors with making changes to the API guidance pages. The general steps to make changes to API guidance are:
+
+- [API Guidance Pages](#api-guidance-pages)
+  - [Start a Github Codespace](#start-a-github-codespace)
+  - [Navigate to/create the Markdown file for the guidance page](#navigate-tocreate-the-markdown-file-for-the-guidance-page)
+  - [Construct the page/contents using supported Markdown syntax](#construct-the-pagecontents-using-supported-markdown-syntax)
+  - [Preview your changes](#preview-your-changes)
+  - [Raise a pull request](#raise-a-pull-request)
+  - [Seek approval/merge your changes](#seek-approvalmerge-your-changes)
+
+## Start a Github Codespace
```

Comment:
The one hurdle here is that we want the CodeSpaces to belong to DfE, rather than be created on the editor's own fork. People need to:

* be in the DFE-Digital GitHub org
* have CodeSpaces enabled for their account

They can check if they're a member by going to [their organisations list](https://github.com/settings/organizations) and lookinf for `DFE-Digital`

![image](https://github.com/DFE-Digital/npq-registration/assets/128088/a76f7950-e263-439a-ace8-8de0f76d25eb)

And when creating a CodeSpace they should see the disclaimer at the bottom of the popup which says 

> Codespace usage for this repository is paid for by DFE-Digital.


![image](https://github.com/DFE-Digital/npq-registration/assets/128088/975ac27e-ec08-49c4-b174-7aa6f36899c5)

I think it's worth mentioning because it's not obvious. I've added a task to my list to try and get CodeSpaces enabled by default for all content designers.